MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.md
 include LICENCE
 include tox.ini
 include requirements.txt
+include requirements-dev.txt
```

Comment:
I don't think it is needed. If I understand correctly, `MANIFEST.in` is used to package end-user versions of Watson, so dev stuff should not added.